src/server/services/afis/afis-route-handlers.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import { fetchAfisBusinessPartner } from './afis';
+
+export async function handleFetchAfisBusinessPartner(
+  req: Request<{ businessPartnerId: string }>,
+  res: Response
+) {
+  const response = await fetchAfisBusinessPartner(
+    res.locals.requestID,
+    req.params.businessPartnerId
+  );
+
+  if (response.status === 'ERROR') {
```

Comment:
Je zou `sendResponse` uit `src/server/helpers/app.ts` kunnen gebruiken in plaats van deze if (precies dezelfde code).